docker/docker.go

```diff
@@ -111,7 +111,7 @@ func main() {
 			*flTls = true
 			cert, err := tls.LoadX509KeyPair(*flCert, *flKey)
 			if err != nil {
-				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted?", err)
+				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted", err)
```

Comment:
@duglin @thaJeztah I have accepted suggestions from both of you, new a PR #11073 to commit the change.
Thx very much.